cli/start.go

```diff
@@ -69,17 +68,24 @@ func (s *StartCommand) runStart(args []string) error {
 			return fmt.Errorf("cannot start and attach multiple containers at once")
 		}
 
-		in, out, err := setRawMode(s.stdin, false)
+		container := args[0]
+		c, err := apiClient.ContainerGet(ctx, container)
 		if err != nil {
-			return fmt.Errorf("failed to set raw mode")
+			return err
 		}
-		defer func() {
-			if err := restoreMode(in, out); err != nil {
-				fmt.Fprintf(os.Stderr, "failed to restore term mode")
+
+		if c.Config.Tty && terminal.IsTerminal(int(os.Stdout.Fd())) {
+			in, out, err := setRawMode(s.stdin, false)
```

Comment:
I think we should return error here if the input/output device is not terminal. How do you think?

Reply:
ok, update.